docs/graphql/manual/migrations/reference/metadata-format.rst

```diff
@@ -25,12 +25,239 @@ of multiple files.
+
+Contains the metadata related to :ref:`remote schemas<remote_schemas>`
+
+..  code-block::  yaml
+
+    - name: local
+      definition:
+        url_from_env: REMOTE_SCHEMA
+        timeout_seconds: 40
+
+
+In above example ``remote_schemas.yaml`` contains information about remote schema local where local graphql urls are specified via
+REMOTE_SCHEMA environment variable.
+
+..  code-block::  yaml
+
+    - name: local
+      definition:
+        url: <graphql_url>
+        timeout_seconds: 40
+
+
+The above example is same to the previous except the url is specified directly not via environment variable.
+
+functions.yaml
+^^^^^^^^^^^^^^
+
+Contains the metadata related to :ref:`custom functions<custom_sql_functions>`
+
+..  code-block::  yaml
+
+    - function:
+      schema: public
+      name: search_books
+
+
+In the above example ``functions.yaml`` file consists of search_books custom sql function
+
+query_collections.yaml
+^^^^^^^^^^^^^^^^^^^^^^
+
+Conatins the information about query query collections
+
+..  code-block::  yaml
+
+  - name: allowed-queries
+    definition:
+      queries:
+      - name: test
+        query: |-
+          query test {
+            books {
+              id
+              author_id
+              title
+            }
+          }
+      - name: test2
+        query: |-
+          query test2 {
+              authors{
+                  id
+                  author_name
+              }
+          }
+    
+In above example there is only one collection ``allowed_queries`` which contains two queries ``test`` and ``test2`` .
+
+
+allow_list.yaml
+^^^^^^^^^^^^^^^
+
+Contains the metadata related to :ref:`allow lists<allow_list>`
+
+..  code-block::  yaml
+
+  - collection: allowed-queries
+
+The allowed queries are under the collection name ``allowed-queries``
+``allow_list.yaml`` contains collection name of allowed queries.
+
+
+
+actions.yaml
+^^^^^^^^^^^^
+
+Contains the metadata related to :ref:`actions<actions>`
+
+..  code-block::  yaml
+
+      actions:
+    - name: greet
+      definition:
+        kind: ""
+        handler: <base_url>/greet
+        forward_client_headers: true
+        headers:
+        - value: application/json
+          name: Content-Type
+    custom_types:
+      enums: []
+      input_objects:
+      - name: SampleInput
+      objects:
+      - name: SampleOutput
+      scalars: []
+
+
+In above example the url is specified directly to handler field
+
+..  code-block::  yaml
+
+      actions:
+    - name: greet
+      definition:
+        kind: ""
+        handler: '{{ACTION_BASE_URL}}/greet'
+        forward_client_headers: true
+        headers:
+        - value: application/json
+          name: Content-Type
+    custom_types:
+      enums: []
+      input_objects:
+      - name: SampleInput
+      objects:
+      - name: SampleOutput
+      scalars: []
+
+
+The above example is same to the previous except the url is specified directly not via environment variable.
```

Comment:
The above example is the same as the previous one except that the URL is specified directly and not via an environment variable.